### PR TITLE
Feature/fetching vumi sent sms count

### PR DESCRIPTION
**Pull request by msinovcic**

### CODE REVIEW

**Comment by dbreedt** [comment]
:+1:

File: gems/core/tasks.py [hunk 1]
```diff
@@ -235,3 +237,65 @@ def sync_group_members(self):
...
+        date_construct_helper(today, timedelta(days=5)),
+        date_construct_helper(today, timedelta(days=4)),
+        date_construct_helper(today, timedelta(days=3)),
+        date_construct_helper(today, timedelta(days=2)),
+        date_construct_helper(today, timedelta(days=1)),
+        date_construct_helper(today, None)
+    ]
+
+    for survey in surveys:
+        metric = 'conversations.' + survey[str('survey_id')] + '.outbound_unique_addresses.avg'
```
Comment (hodgestar) [inline gems/core/tasks.py:270]: This calls `str()` on something that is already a string?

File: gems/core/tasks.py [hunk 2]
```diff
@@ -235,3 +237,65 @@ def sync_group_members(self):
...
+        logger.info('Fetching metric: ' + metric + ' for channel: ' + survey[str('survey_id')])
```
Comment (hodgestar) [inline gems/core/tasks.py:271]: Same here.

File: gems/core/tasks.py (hunk 1 above)
Comment (dbreedt) [inline gems/core/tasks.py:270]: @hodgestar: survey['survey_id']: dictionary doesn't contain a key called 'survey_id', hence the reason for the str() cast.

Comment (hodgestar) [inline gems/core/tasks.py:270]: Still not seeing why `str()` would be needed -- `str('survey_id') is
'survey_id'`.

**Pull request merged**